contents/handbook/product/user-feedback.md

```diff
@@ -4,67 +4,52 @@ sidebar: Handbook
-- The topics of the conversation are usually better informed from the previous step. It's always more helpful to talk about topics where the user has had more experience. It's however always useful to consider what we're currently working on at Product/Growth/Marketing and ask questions that could help each team do a better job.
-
 
 **After the call**
-1. Upload the recording to the [recordings folder][recordings], be sure to follow naming conventions.
-2. Add the notes to the [Google Doc][feedback-doc], linking the recording. Try to keep them as clean as possible.
-3. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Orbit and/or Hubspot record in case Engineering needs more context when scoping/building.
-4. Generate the reward for the user (see below).
-   1. Most of the time, the reward will be a gift card for the [PostHog merch store](https://merch.posthog.com/). If it's the case, create the gift card in Shopify. Make sure to **add a link to their Orbit profile** in the notes section and set the **default expiration date** (5 years), and make note of the gift card code.
-5. Register call on Orbit. Remember to link the notes, add appropriate labels (e.g. `received merch`) and information on the rewards sent.
-6. If the customer is Sales/Scale, register the call on Hubspot too (but linking to Google doc notes).
-7. Follow-up with the user. Send any applicable rewards, links to any opened GitHub issues, and answers to any outstanding questions.
-
+1. Add the notes to the [Google Doc][feedback-doc], linking the recording if you did one.
+2. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Hubspot record in case Engineering needs more context when scoping/building.
+3. Generate the reward for the user (see below).
+   1. Most of the time, the reward will be a gift card for the [PostHog merch store](https://merch.posthog.com/). If it's the case, create the gift card in Shopify. Make sure to **add a link to their Hubspot profile** in the notes section and set the **default expiration date** (5 years), and make note of the gift card code.
+4. Follow-up with the user. Send any applicable rewards, links to any opened GitHub issues, and answers to any outstanding questions.
 
 ### Rewards 
-We strongly value our users' time. We particularly acknowledge the value it provides us in building a better product. As such, we usually send a small gift of appreciation. We have the following general _guidelines_, but just use your best judgement.
-- Generally we send users a gift card to the merch store with around $50 of value.
+We strongly value our users' time. As such, we usually send a small gift of appreciation. We have the following general _guidelines_, but just use your best judgement.
+- Generally we send users a gift card to the merch store with around $30 of value.
```

Comment:
@lharries You changed the amount from $50 to $30. I believe the user interview popup still says $50? If so, let me know once you've updated it and I know what to pay users before/after the change.

Reply:
I think $30 makes more sense? I'll update the user interview app

Reply:
Done

Reply:
I think $30 is a good amount, especially with the options we have in the merch store 👍